src/js/disability-benefits/526EZ/components/DisabilityWizard.jsx

```diff
@@ -0,0 +1,191 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import appendQuery from 'append-query';
+
+import RequiredLoginView from '../../../common/components/RequiredLoginView';
+import ErrorableCheckboxes from '../../../common/components/form-elements/ErrorableCheckboxes';
+import ErrorableRadioButtons from '../../../common/components/form-elements/ErrorableRadioButtons';
+
+import { toggleLoginModal } from '../../../login/actions';
+
+import { disabilityStatusOptions, disabilityUpdateOptions, layouts } from '../helpers';
+
+const { chooseStatus, chooseUpdate, applyGuidance } = layouts;
+
+class DisabilityWizard extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      currentLayout: chooseStatus
+    };
+  }
+
+    ButtonContainer = () => {
+      const { currentLayout, disabilityStatus, add, increase } = this.state;
+      const { profile, loginUrl, verifyUrl } = this.props;
+      const notUpdatingStatus = disabilityStatus === 'first' || disabilityStatus === 'appeal';
+      const updatingStatus = disabilityStatus === 'update';
+      const eligibleForIncrease = updatingStatus && !add && increase;
+      const ineligibleForIncrease = updatingStatus && add;
+      const atIncreaseGuidance = currentLayout === applyGuidance && eligibleForIncrease;
+      const atEbenefitsGuidance = currentLayout === applyGuidance && (notUpdatingStatus || ineligibleForIncrease);
+
+      return  (<div>
+        {currentLayout !== chooseStatus &&
+        <button type="button" className="usa-button-secondary" onClick={this.goBack}><span className="button-icon">« </span>Back</button>
+        }
+        {atIncreaseGuidance && !sessionStorage.userToken &&
+        <a className="usa-button-primary" href="/disability-benefits/526/apply-for-increase/introduction/" onClick={this.authenticate}>Sign in<span className="button-icon"> »</span></a>
+        }
+        {atIncreaseGuidance && sessionStorage.userToken &&
+        <RequiredLoginView
```

Comment:
:thinking: Seems odd to see this in a `<RequiredLoginView>`, but it works. :man_shrugging: 
:+1: